.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ name: Build-Test-Deploy
       - opened
       - synchronize
       - reopened
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
Shouldn't this also target a branch? Otherwise this would end up running for all branches with this workflow on it.

Reply:
No, schedule only works on the "default" branch, which happens to be "master" for us.